backend/src/test/java/com/soen341/instagram/dao/impl/AccountRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package com.soen341.instagram.dao.impl;
+
+import com.soen341.instagram.model.Account;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class AccountRepositoryTest {
+    @Autowired
+    private AccountRepository accountRepository;
+
+    private Account account;
+
+    @BeforeEach
+    public void beforeEach(){
+        accountRepository.deleteAll();
```

Comment:
I don't think you need this because `@DataJpaTest` makes the class Transactional. So, after every tests, everything gets rolled back by default. Can you try it and get back to me.

Reply:
Same thing in `CommentRepositoryTest`

Reply:
If there's something that handles that by default, I think it'd be great. I know for the tests I wrote during my internship I occasionally had to use `beforeEach()` and `afterEach()` but if it's done by default and we could reduce/eliminate redundant code, that'd be great 👍